corehq/apps/hqcase/tests/test_case_update_api.py

```diff
@@ -14,13 +13,17 @@
     CaseAccessors,
     FormAccessors,
 )
-from corehq.form_processor.tests.utils import FormProcessorTestUtils
+from corehq.form_processor.tests.utils import (
+    FormProcessorTestUtils,
+    use_sql_backend,
+)
 from corehq.util.test_utils import privilege_enabled
 
 from ..api import serialize_case
 from ..utils import submit_case_blocks
 
 
+@use_sql_backend
```

Comment:
Should this use `@run_with_all_backends`?

Reply:
While I did just add in support for couch, I think the answer here is no, in the interest of test performance.  It's generally assumed that functionality works on both backends, but we only explicitly test both for dbaccessors, otherwise we'd be running _all_ tests on both backends.  Since the dbaccessors used are tested individually on their respective backends, I think it's safe to only test this on one.  Since SQL is the one I care about more, I do think it makes sense to keep this here, though I did confirm that tests pass locally on both backends.  Would you agree with that reasoning?